hs_discover/views.py

```diff
@@ -135,41 +138,37 @@ def get(self, request, *args, **kwargs):
             return JsonResponse({'message': '{}'.format('{}: query error. Contact a server administrator.'
                                                         .format(type(gen_ex)))}, status=520)
 
+        filterdata = []
+        if request.GET.get('filterbuilder'):
```

Comment:
ok, just so I can keep this straight.  First pass, you had filterbuilder with it's own response skipping the rest of the logic in this endpoint.  Second pass, you included it with all responses.  Now, this third pass it's being included only when requested?

Reply:
If by pass you mean versions of this endpoint that you've reviewed, then that's generally correct. Initially, you saw a version of filterbuilder that was called in a separate axios call and the filters built, in parallel (non blocking) operations alongside the resource listing. Now, since the index search will impact the facet counts, it is performed inline with the endpoint logic.